challenge.rb

```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.collect do |word|
```

Comment:
nice use of collect which is an alias for `.map`.

you could also write this `input.map(&:capitalize)` or `input.map {|s| s.capitalize}`.

be sure to take a look at the Symbol#to_proc method. it's very common.
https://ruby-doc.org/core-1.9.3/Symbol.html#method-i-to_proc